### PR TITLE
Add support for Italian language

### PR DESCRIPTION
**Pull request by andreeaceachir142**
# What 

- Add support for Italian language 

# How 

- Add new `it.js` file with all the Italian translations
- Add Italian layout for the keyboard

### CODE REVIEW

**Comment by ammapspeople** [comment]
Remember to update the descriptions for the language prop.
Also on the docs site.

**Comment by andreeaceachir142** [comment]
Done ✅ @ammapspeople

File: packages/map-template/.github/README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ When loading the MapsIndoors Map component for the first time, the map will resp
 |`pitch`|`number`|The pitch (Mapbox) or tilt (Google) value of the map. The value of the pitch can be between 0-85 degrees on a Mapbox map. |
 |`bearing`|`number`|The bearing (Mapbox) or heading (Google) value of the map. It accepts any value, and will modify it to fit into the range [0, 360]. |
 |`useMapProviderModule`|`bool`|Set to `true` if the Map Template should take MapsIndoors Solution Modules into consideration when determining what map type to use.|
-|`language`|`string`|The language to show textual content in. Supported values are "en" for English, "da" for Danish, "de" for German and "fr" for French. If the prop is not set, the language of the browser will be used (if it is one of the four supported languages - otherwise it will default to English). |
+|`language`|`string`|The language to show textual content in. Supported values are "en" for English, "da" for Danish, "de" for German, "fr" for French and "it" for Italian. If the prop is not set, the language of the browser will be used (if it is one of the four supported languages - otherwise it will default to English). |
```
[the remark this answers is not in the document]
Comment (andreeaceachir142) [reply packages/map-template/.github/README.md]: ✅

**Comment by matbmapspeople** [comment]
LGTM ✅ I could not find anything while testing for both: MT and Kiosk @andreeaceachir142

**Comment by andreeaceachir142** [comment]
Closing this off as it was released together with the Spanish language.

**Pull request merged**